src/screens/Dashboard/index.tsx

```diff
@@ -1,30 +1,62 @@
 import React from 'react';
 import {createStackNavigator, TransitionPresets} from '@react-navigation/stack';
-import Dashboard from './Dashboard';
 import TripSearch from '@atb/screens/Dashboard/TripSearch';
+import {TripDetailsRoot} from '../TripDetails';
+import {SearchTime} from '../Nearby/types';
+import {Location} from '@atb/favorites/types';
+import {DetailsRouteParams} from '../TripDetails/Details';
+import JourneyDatePicker, {
+  DateTimePickerParams,
+} from '../Assistant/journey-date-picker';
```

Comment:
Same as the other stuff I think this should be duplicated in the transition/feature toggle phase.